### ISSUE TITLE
Clicking on "News" in top nav directs to /Updates

### ISSUE THREAD

**Issue by TylerRadford** [hotosm/hotosm-website#295]
Hi, can we match up the top nav menu items with their associated directories?
Right now clicking "News" brings you to /Updates instead of /News
![image](https://user-images.githubusercontent.com/9853903/41246128-1a61600e-6d78-11e8-99fb-647b4428fa9a.png)

**Comment by brendangatens** [comment]
We had a discussion about this a while back and ended up going with **News** as the most appropriate label and nav item for this section. All blog posts on the old site live under the `/updates` directory, however. So before changing the url to `/news` we'll have to create redirects from `/updates`, and inform search engines of the change.

**Comment by smit1678** [comment]
Yeah, we had flagged this back in March as a item to think about https://github.com/hotosm/hotosm-website/issues/168

**Comment by Muskan284** [comment]
@smit1678 @ramyaragupathy can i work on this issue

**Comment by ramyaragupathy** [comment]
@smit1678 - what do you think about updating the ln update here?
